src/Microsoft.HttpRepl/Commands/UICommand.cs

```diff
@@ -17,6 +16,17 @@ namespace Microsoft.HttpRepl.Commands
     public class UICommand : ICommand<HttpState, ICoreParseResult>
     {
         private static readonly string Name = "ui";
+        private IUriLauncher UriLauncher;
+
+        public UICommand(IUriLauncher uriLauncher)
+        {
+            if (uriLauncher == null)
```

Comment:
you can one-line this, which is pretty common style now:
`UriLauncher = uriLauncher ?? throw new ArgumentNullException(nameof(uriLauncher));`

Reply:
Updated here:
https://github.com/aspnet/HttpRepl/commit/ff18a5d8cac490242498c8397f068261b1eb7966